test/integration_test/storkctl_dr_action_test.go

```diff
@@ -0,0 +1,312 @@
+		"migration-reference": migrationScheduleName,
+		"include-namespaces":  defaultNs,
+		"namespace":           namespace,
+	}
+	expectedAction := generateDRActionObject(storkv1.ActionTypeFailback, migrationScheduleName, []string{defaultNs}, false)
+	createDRActionTest(t, testrailId, cmdArgs, expectedAction, migrationScheduleName, asyncDR, actionType, namespace)
+}
+
+func createFailbackActionWithExcludeNsTest(t *testing.T) {
+	testrailId := 296304
+	actionType := "failback"
+	// the dummy migrationSchedule created migrates defaultNs and adminNs
+	migrationScheduleName := "test-action-automation-migration-schedule"
+	namespace := adminNs
+	cmdArgs := map[string]string{
+		"migration-reference": migrationScheduleName,
+		"exclude-namespaces":  adminNs,
+		"namespace":           namespace,
+	}
+	expectedAction := generateDRActionObject(storkv1.ActionTypeFailback, migrationScheduleName, []string{defaultNs}, false)
+	createDRActionTest(t, testrailId, cmdArgs, expectedAction, migrationScheduleName, asyncDR, actionType, namespace)
+}
+
+func createDRActionTest(t *testing.T, testrailID int, args map[string]string, expectedAction *storkv1.Action, migrationScheduleName string,
+	clusterPairMode string, actionType string, namespace string) {
+	var testResult = testResultFail
+	runID := testrailSetupForTest(testrailID, &testResult, t.Name())
+	defer updateTestRail(&testResult, testrailID, runID)
+	defer migrationScheduleCleanup(t, migrationScheduleName, namespace)
+	defer updateDashStats(t.Name(), &testResult)
+
+	// Create the required migrationschedule in the given namespace
+	createMigrationSchedule(t, migrationScheduleName, namespace, clusterPairMode)
+	// Create the DR Action in the given namespace and verify output
+	drActionName, _ := createDRAction(t, namespace, actionType, migrationScheduleName, args)
+	// validate the action object created
+	err := ValidateAction(t, expectedAction, drActionName, namespace)
+	log.FailOnError(t, err, "Error validating the created resource")
+
+	// try to get the status as well
+	_, _, _ = getDRActionStatus(t, actionType, drActionName, namespace)
+	// Cleanup the created action resource
+	err = storkops.Instance().DeleteAction(drActionName, namespace)
+	log.FailOnError(t, err, "Error deleting the created action resource")
+	// If we are here then the test has passed
+	testResult = testResultPass
+	log.InfoD("Test status at end of %s test: %s", t.Name(), testResult)
+}
+
+func createDRAction(t *testing.T, namespace string, actionType string, migrationScheduleName string, customArgs map[string]string) (string, []string) {
+	factory := storkctl.NewFactory()
+	var outputBuffer bytes.Buffer
+	cmd := storkctl.NewCommand(factory, os.Stdin, &outputBuffer, os.Stderr)
+	cmdArgs := []string{"perform", actionType, "--kubeconfig", kubeConfigPath}
+	executeStorkCtlCommand(t, cmd, cmdArgs, customArgs)
+	// Get the captured output as a string
+	actualOutput := outputBuffer.String()
+	log.InfoD("Actual output is: %s", actualOutput)
```

Comment:
print command and then output of it.

Reply:
Makes sense. Will address

Reply:
The command being executed is printed already as part of the `executeStorkCtlCommand` method